app/design/adminhtml/default/openmage/template/login.phtml

```diff
@@ -0,0 +1,77 @@
+
+    <script type="text/javascript" src="<?php echo $this->getJsUrl('prototype/prototype.js') ?>"></script>
+    <script type="text/javascript" src="<?php echo $this->getJsUrl('prototype/validation.js') ?>"></script>
+    <script type="text/javascript" src="<?php echo $this->getJsUrl('scriptaculous/effects.js') ?>"></script>
+    <script type="text/javascript" src="<?php echo $this->getJsUrl('mage/adminhtml/form.js') ?>"></script>
+    <script type="text/javascript" src="<?php echo $this->getJsUrl('mage/captcha.js') ?>"></script>
+
+    <!--[if IE]> <link rel="stylesheet" href="<?php echo $this->getSkinUrl('iestyles.css') ?>" type="text/css" media="all" /> <![endif]-->
+    <!--[if lt IE 7]> <link rel="stylesheet" href="<?php echo $this->getSkinUrl('below_ie7.css') ?>" type="text/css" media="all" /> <![endif]-->
+    <!--[if IE 7]> <link rel="stylesheet" href="<?php echo $this->getSkinUrl('ie7.css') ?>" type="text/css" media="all" /> <![endif]-->
+</head>
+<body id="page-login" onload="document.forms.loginForm.username.focus();">
+    <div class="login-container">
+        <div class="login-box">
+            <form method="post" action="" id="loginForm" autocomplete="on">
+                <div class="login-form">
+                    <input name="form_key" type="hidden" value="<?php echo $this->getFormKey() ?>" />
+                    <h2><?php echo Mage::helper('adminhtml')->__('Log in to Admin Panel') ?></h2>
+                    <div id="messages">
+                        <?php echo $this->getMessagesBlock()->toHtml() ?>
+                    </div>
+                    <div class="input-box"><label for="username"><?php echo Mage::helper('adminhtml')->__('User Name:') ?></label><br/>
+                        <input type="text" id="username" name="login[username]" value="" class="required-entry input-text" /></div>
+                    <div class="input-box"><label for="login"><?php echo Mage::helper('adminhtml')->__('Password:') ?></label><br />
+                        <input type="password" id="login" name="login[password]" class="required-entry input-text" value="" /></div>
+                    <?php echo $this->getChildHtml('form.additional.info'); ?>
+                    <div class="clear"></div>
+                    <div class="form-buttons">
+                        <a class="left" href="<?php echo Mage::helper('adminhtml')->getUrl('adminhtml/index/forgotpassword', array('_nosecret' => true)) ?>"><?php echo Mage::helper('adminhtml')->__('Forgot your password?') ?></a>
+                        <input type="submit" class="form-button" value="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" title="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" /></div>
+                </div>
+                <p class="legal"><?php echo Mage::helper('adminhtml')->__('OpenMage - Actively supported fork of Magento Community Edition') ?></p>
```

Comment:
Please just call it OpenMage LTS and do not mention Magento CE. See discussion in https://github.com/OpenMage/magento-lts/pull/1014